src/constants/filterConfigs.js

```diff
@@ -31,15 +31,38 @@ export const titleSortFilterConfig = {
   ],
 };
 
+export const selectionStatusFilterOptions = {
+  ALL: 'all',
+  TRUE: 'true',
+  FALSE: 'false',
+  EBSCO: 'ebsco',
+};
+
+export const ebscoProviderId = 19;
```

Comment:
For constant variables for number, better to name them in uppercase
```suggestion
export const EBSCO_PROVIDER_ID = 19;
```

Reply:
Fixed